js/profile.js

```diff
@@ -16,6 +16,9 @@ $("document").ready(function() {
     context.has_publications = function(){
       return this.publications.length > 0;
     }
+    context.has_other_publications = function(){
+      return (typeof this.other_publications != "undefined");
+    }
```

Comment:
So the purpose of this logic is so that we can determine whether or not to show a tab for this data in the profile based on whether or not there's any data to show. If it's just going to be empty, then we don't show the tab at all. However, it looks to me like our API is always going to return an object for `other_publications`, so it'll never be undefined. If that's the case, then I think we're going to end up aways displaying this tab even if there's nothing to show in it (i.e. even when the `other_publications` object is empty. Am I reading that right?

If that's the case, then we might want a different check here like `this.other_publications.isEmpty` or something like that.

This same idea applies to the subheadings within other_publications, but it looks like you handled that nicely over in [RMD](https://github.com/psu-stewardship/researcher-metadata/blob/897ec5667c7de8ad4dae8208f831cd20495507e6/app/models/user_profile.rb#L134)

Reply:
Nice catch.  I'll change this.